src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigureDraw {
+    private String color;

```

Comment:
I would declare this field as `final`, since a figure cannot exist without a color